src/dotnet-interactive-vscode/src/tests/unit/notebook.test.ts

```diff
@@ -112,29 +121,36 @@ Console.WriteLine(1);
                                 value: '3\r\n'
                             }
                         ]
-                    }
+                    },
+                    token
                 },
                 {
                     eventType: 'CommandHandled',
-                    event: {}
+                    event: {},
+                    token
                 }
             ]
         }));
         let client = clientMapper.getOrAddClient({ path: 'test/path' });
-        let outputs = await execute('csharp', code, client);
-        expect(outputs).to.deep.equal([
-            {
-                outputKind: CellOutputKind.Text,
-                text: '1\r\n'
-            },
-            {
-                outputKind: CellOutputKind.Text,
-                text: '2\r\n'
-            },
-            {
-                outputKind: CellOutputKind.Text,
-                text: '3\r\n'
+        let outputs: Array<CellOutput> = [];
+        await client.execute('csharp', code, cellOutput => {
```

Comment:
usually we follow the convention `(code:string, targetKenelName: string)` is this our type or vscode interace?

Reply:
This is our own interface; I'll reverse it to match.